rtengine/ipdehaze.cc

```diff
@@ -229,100 +269,143 @@ void ImProcFunctions::dehaze(Imagefloat *img)
 
     int patchsize = max(int(5 / scale), 2);
     float ambient[3];
-    array2D<float> &t_tilde = dark;
     float max_t = 0.f;
```

Comment:
`max_t` is a bad variable name, as `_t` usually denotes a type.

Reply:
Changed